js/app.js

```diff
@@ -126,6 +166,7 @@
    *   and passing two aditional arguments, event and 'light-green'
    */
 
+   btn9.addEventListener("click", (event) => setMyLightClass.call(btn9, event, "light-green"));
```

Comment:
Because we only have one parameter, the fat arrow could be written without the parenthesis. This is more of a FYI then a change request. This also applies to any previous fat arrow declarations previous to this line of code.